checker/src/org/checkerframework/checker/fenum/FenumAnnotatedTypeFactory.java

```diff
@@ -34,6 +35,15 @@ public FenumAnnotatedTypeFactory(BaseTypeChecker checker) {
         this.postInit();
     }
 
+    @Override
+    protected void addCheckedCodeDefaults(QualifierDefaults defs) {
+        // An error is issued if the type of a thrown exception
+        // is not @FenumUnqualified.
+        defs.addCheckedCodeDefault(FENUM_UNQUALIFIED, DefaultLocation.EXCEPTION_PARAMETER);
```

Comment:
@FenumUnqualified is annotated with @DefaultQualifierInHeirarchy, so it cannot also have @DefaultFor(EXCEPTION_PARAMETER).  This code is required in order to override the CLIMB-to-the-top defaults.  If you like this design, I'll add a comment.  

Alternatively, we could allow  @DefaultQualifierInHeirarchy and  @DefaultFor on the same annotation.  Or force type systems to use every CLIMB default or explicitly state all defaults.